charts/ark-activemq/templates/statefulset.yaml

```diff
@@ -9,17 +9,18 @@ spec:
   selector:
     matchLabels:
       {{- include "ark-activemq.selectorLabels" . | nindent 6 }}
+  serviceName: {{ include "ark-activemq.fullname" . }}
```

Comment:
@fabricetriboix For a statefulset we would require a headless service. Looks service manifest file is not updated with the latest commits. Could you check it.

Reply:
Yes, that's right, I forgot that. I will do it now.